lib/api/src/env.rs

```diff
@@ -76,6 +76,10 @@ pub trait WasmerEnv: Clone + Send + Sync {
     fn init_with_instance(&mut self, _instance: &Instance) -> Result<(), HostEnvInitError> {
         Ok(())
     }
+
+    #[ cfg(feature="async") ]
+    /// Set the thread-specific yielder
+    fn set_yielder(&mut self, _: *const std::ffi::c_void) {}
```

Comment:
This probably needs to be `unsafe` and have a `# Safety` section describing how to correctly call and/or implement this function